### DISCUSSION TITLE
chia version 1.2.9 don't open (error 940195/ubuntu)

### DISCUSSION THREAD

**Discussion by routerlocal** [Chia-Network/chia-blockchain#8683] [Support]
i open the chia gui and this return error 940195

When opening the chia this error appears, then the "fingerprint" are displayed, after click to load, the loading loader always and back to choose the finger print.

![bug](https://user-images.githubusercontent.com/84590680/135717273-8ed1acf6-688f-43f2-a7e6-228fb566be3f.png)

**Comment by loppefaaret** [comment]
if you are still facing this problem, could you try and look into your `debug.log` file located under your users folder `~/.chia/mainnet/log/` and look for ERROR entries from the `full_node` ?

**Reply by routerlocal** [reply]
Sorry, but this was 5 days ago, as I didn't get help before, I ended up formatting the farm and recreating it from scratch.

Grateful!

**Comment by BrandtH22** [comment]
@routerlocal , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
